memory/ndctl_selftest.py

```diff
@@ -45,6 +45,39 @@ def get_bus_ids(self):
                     bus_ids.append(value)
         return bus_ids
 
+    def autotools_build_system(self):
```

Comment:
I suggest  patch[04/03] and patch[04/05] can merge with this patch itself as having broader commit as meson build support

Reply:
Done